dsl/docker-multiarch/images/openjdk.groovy

```diff
@@ -0,0 +1,44 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/docker-library/openjdk.git') }
+				branches('*/master')
+				clean()
+			}
+		}
+		triggers {
+			upstream("docker-${arch}-debian, docker-${arch}-buildpack", 'UNSTABLE')
+			scm('H H/6 * * *')
+		}
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+case "$dpkgArch" in
+	ppc64el)
+		sed -i 's!amd64!ppc64el!g' */Dockerfile
+		sed -i 's!unzip!unzip udev!' openjdk-9-jdk/Dockerfile
+esac
+
+sed -i "s!^FROM !FROM $prefix/!" */Dockerfile
```

Comment:
I'd prefer to move this line to the top of the `script` block here just so we have symmetry with other files here (https://github.com/tianon/jenkins-groovy/blob/b11361e40e555e23f0d64be118b25433ca1d18b7/dsl/docker-multiarch/images/gcc.groovy#L23-L25 for example).